packages/python/BUILD

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2015 Nuxi, https://nuxi.nl/
+#
+# This file is distributed under a 2-clause BSD license.
+# See the LICENSE file for details.
+
+import os
+import sysconfig
+
+def build(ctx):
+    srcdir = ctx.extract('Python-3.6.0a2')
+
+    # Delete the in-tree copy of these, to make certain the out-of-tree
+    # versions are used, i.e. those generated from our patched importlib
+    os.unlink(os.path.join(str(srcdir), 'Python', 'importlib.h'))
+    os.unlink(os.path.join(str(srcdir), 'Python', 'importlib_external.h'))
+
+    # First build a native binary, this will be used by the cross-build
+    # to generate some files (see also patch-makefile-regen)
+    hostbuild = srcdir.host().gnu_configure()
+    hostbuild.make()
+
+    build = srcdir.gnu_configure([
+        '--build=%s' % sysconfig.get_config_var('HOST_GNU_TYPE'),
```

Comment:
I'm a bit in doubt what to do here. Some time ago, I added `--build` to the configure flags in `src/builder.py`, but I noticed that this then trickles into the package contents (e.g., part of `bin/...-config`). Maybe the right thing to do would be to not pass this in as a flag, but add a parameter to the function: `gnu_configure(add_build=True)`. I'd rather not solve all of this in this package specifically.

What are your thoughts?